src/controller/node.c

```diff
@@ -76,6 +76,8 @@ static const sd_bus_vtable node_vtable[] = {
         SD_BUS_METHOD("ThawUnit", "s", "", node_method_passthrough_to_agent, 0),
         SD_BUS_METHOD("RestartUnit", "ss", "o", node_method_restart_unit, 0),
         SD_BUS_METHOD("ReloadUnit", "ss", "o", node_method_reload_unit, 0),
+        SD_BUS_METHOD("ResetFailed", "", "", node_method_passthrough_to_agent, 0),
```

Comment:
I think that if we are adding `bluechictl reset-failed` functionality (on all nodes) - it makes sense to enable that in the controller. 
1. Saves request to the controller when doing that
2. Makes sense as a feature for controller as well

@engelmi wdyt?

Reply:
Should go in here
https://github.com/eclipse-bluechi/bluechi/blob/a890cbbb9b6c3f806c1e2cb91b3c686df88ef7e3/src/controller/controller.c#L1179
Not completely necessary though, `bluechictl` can fetch node list

Reply:
As discussed in a call, we will not proceed with adding `ResetFailed` to the controller API, as it is used as system management API and not for actions on all nodes.